lib/parser.js

```diff
@@ -188,7 +198,20 @@ module.exports = function parser(identifier, string) {
   // Parses field options: [ name1 = value1, name2 = value2 ]
   function parseFieldOptions(parent) {
     parseBlock(function (token) {
+      var hasParens = false
+      var nameToken = token
+      if (token.type == Token.Type.START_PAREN) {
```

Comment:
sadly, i don't think the protocol buffer language has an official grammar? i found an unofficial one but it's missing some pieces:
https://groups.google.com/forum/#!topic/protobuf/HMz8YkzWEto

here's the parser code that handles this case:
https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/parser.cc#L1079